src/modules/Channel/context/ChannelProvider.tsx

```diff
@@ -162,6 +162,12 @@ export interface ChannelProviderInterface extends ChannelContextProps, MessageSt
 
 const ChannelContext = React.createContext<ChannelProviderInterface | null>(null);
 
+/**
+ * @deprecated This provider is deprecated and will be removed in future releases.
```

Comment:
I think `next major update` is better than `future release`. (`future release` is too general to determine a specific time.)